pennylane/ops/functions/__init__.py

```diff
@@ -19,19 +19,21 @@
 .. autosummary::
     :toctree: api
 
+    ~bind_parameters

```

Comment:
```suggestion
    ~ops.functions.bind_parameters
```

Current module is `pennylane`, not `ops.functions`.  Might be fine, but I think this might be necessary.